libraries/abstractions/ble_hal/include/bt_hal_manager_adapter_ble.h

```diff
@@ -97,8 +97,15 @@ typedef struct
     BTGattAdvName_t ucName;
     bool bSetScanRsp;
     uint32_t ulAppearance;
-    uint32_t ulMinInterval; /**< Minimum Connection Interval. If set to 0, minimum connection interval is not included in advertisement/scan response data. */
-    uint32_t ulMaxInterval; /**< Maximum Connection Interval. If set to 0, maximum connection interval is not included in advertisement/scan response data. */
+    uint32_t ulMinInterval;    /**< Minimum Connection Interval. If set to 0, minimum connection interval is not included in advertisement/scan response data. */
+    uint32_t ulMaxInterval;    /**< Maximum Connection Interval. If set to 0, maximum connection interval is not included in advertisement/scan response data. */
+    uint16_t usMinAdvInterval; /**< Minimum Advertising Interval in units of 0.625ms.
```

Comment:
The interval units is per either an open standard or RFC?

Reply:
This units (and in general the HAL ) should be as per the BLE standard spec.